dags/bqetl_kpi_dashboard.py

```diff
@@ -0,0 +1,59 @@
+# Generated via query_scheduling/generate_airflow_dags
+
+from airflow import DAG
+from airflow.operators.sensors import ExternalTaskSensor
+import datetime
+from utils.gcp import bigquery_etl_query
+
+default_args = {
+    "owner": "jklukas@mozilla.com",
+    "start_date": datetime.datetime(2020, 5, 12, 0, 0),
+    "email": ["telemetry-alerts@mozilla.com", "jklukas@mozilla.com"],
+    "depends_on_past": False,
+    "retry_delay": datetime.timedelta(seconds=600),
+    "email_on_failure": True,
+    "email_on_retry": True,
+    "retries": 1,
+}
+
+with DAG(
+    "bqetl_kpi_dashboard", default_args=default_args, schedule_interval="45 15 * * *"
+) as dag:
+
+    telemetry_derived__smoot_usage_new_profiles__v2 = bigquery_etl_query(
+        task_id="telemetry_derived__smoot_usage_new_profiles__v2",
+        destination_table="smoot_usage_new_profiles_v2",
+        dataset_id="telemetry_derived",
+        project_id="moz-fx-data-shared-prod",
+        owner="jklukas@mozilla.com",
+        email=["jklukas@mozilla.com"],
+        depends_on_past=False,
+        dag=dag,
+    )
+
+    telemetry_derived__smoot_usage_new_profiles_compressed__v2 = bigquery_etl_query(
```

Comment:
These generated task IDs seem a bit unwieldy. Perhaps we should allow specifying a task ID as part of the `scheduling` metadata.

Reply:
Yeah, it might be nice to optionally specify a task name. I think it might be best to add this to a separate PR to avoid bloating this PR even more. There is some additional validation that needs to be done (e.g. making sure task names are unique within one DAG).